crates/language_tools/src/lsp_log_view.rs

```diff
@@ -632,17 +636,24 @@ impl LspLogView {
                     .or_else(move || {
                         let capabilities =
                             lsp_store.lsp_server_capabilities.get(&server_id)?.clone();
-                        let name = lsp_store
-                            .language_server_statuses
-                            .get(&server_id)
-                            .map(|status| status.name.clone())?;
+                        let language_server_status =
+                            lsp_store.language_server_statuses.get(&server_id)?;
+
                         Some(ServerInfo {
                             id: server_id,
                             capabilities,
-                            binary: None,
-                            name,
-                            workspace_folders: Vec::new(),
-                            configuration: None,
+                            name: language_server_status.name.clone(),
```

Comment:
I know `LanguageServerStatus` stores a bit more info, but code-wise, it seems simpler to store that one struct as one field?

Should we try that?